cnf-tests/testsuites/pkg/utils/reporter.go

```diff
@@ -92,6 +94,7 @@ func NewReporter(reportPath string) (*k8sreporter.KubernetesReporter, error) {
 		{Cr: &ocpbuildv1.BuildConfigList{}, Namespace: &namespaces.SroTestNamespace},
 		{Cr: &ocpbuildv1.BuildList{}, Namespace: &namespaces.SroTestNamespace},
 		{Cr: &netattdefv1.NetworkAttachmentDefinitionList{}},
+		{Cr: &metallbv1beta1.MetalLBList{}},
```

Comment:
you also need to add the scheme here https://github.com/openshift-kni/cnf-features-deploy/pull/1212/files#diff-1187a9bc7c5f7569a86f8677f5cdd3edba0b2eb90251d22208cbbec8de28f58dL40